AprilTagTrackers/CMakeLists.txt

```diff
@@ -3,9 +3,9 @@
 #
 cmake_minimum_required (VERSION 3.8)
 
-if(NOT EXISTS ${APRILTAG_LIB} )
-  message(FATAL_ERROR "Please set the correct apriltags library." )
-endif()
+#if(NOT EXISTS ${APRILTAG_LIB} )
+#  message(FATAL_ERROR "AprilTag lib does not exist at ${APRILTAG_LIB}" )
+#endif()

```

Comment:
Why is this commented out?

Reply:
It isn't needed now that the build process is consolidated into one (rather than manually building apriltags separately).